minhphan156/Abode
# SJSU-1048_1, Change Reservation Frontend

Change Reservation dialog on /history page, with details about current and new reservation, and associated refund/payment.

To test: 
1. Click on **CHANGE** button on **/history**
2. Use Calendar picker to pick new reservation dates.
3. Click **Select Dates** button. It will show you new reservation details and associated refund/payment if relevant.
4. Repeat 2-3 to edit dates.
5. Click **Confirm Change** button. Check for updated prices on Expansion panel for the changed booking on /history.
